package.json

```diff
@@ -61,5 +61,19 @@
       "prod",
       "resolutions"
     ]
+  },
+  "pnpm": {
```

Comment:
Do we need this on top of the `peerDependencies` change?

Reply:
It is just for my local test. After the tsp next version released, I'll remove it.